examples/resources/elasticstack_kibana_alerting_rule/resource-index-rule.tf

```diff
@@ -0,0 +1,35 @@
+provider "elasticstack" {
```

Comment:
Not sure if we should add the provider block to each file as it would throw an error if doing so. What about a common/global reference to it (maybe in prerequisites)?

Maybe the intention is just to let the user know that they'll have to add it somewhere, but then we should add the kibana property here too as without it the user won't be able to use a Kibana API like the rules API

Reply:
I've removed the repeated provider blurb via b10a90b5be59f3f263201fb398431282b83c348c and also added a comment in the prerequisites section saying that you need to already have your provider set up for your cluster or deployment.